core/simpleproducer.go

```diff
@@ -123,6 +125,19 @@ func (prod *SimpleProducer) Configure(conf PluginConfigReader) error {
 	dropStreamID := StreamRegistry.GetStreamID(conf.GetString("DropToStream", DroppedStream))
 	prod.dropStream = StreamRegistry.GetRouterOrFallback(dropStreamID)
 
+	// Simple health check for the plugin state
+	//   Path: "/<plugin_class>/<plugin_id>/SimpleProducer/pluginstate"
```

Comment:
I think exposing each health check behind its own URL path gives more granularity for the user; they can e.g. give different priorities to different checks or choose to ignore some. OTOH, the /_ALL path still lets you run all checks at once. 

Skipping /<plugin_class> makes sense, although it could be informative to show also what type of plugin each check belongs to.